components/ai_chat/core/browser/conversation_handler.cc

```diff
@@ -1685,7 +1685,7 @@ void ConversationHandler::OnClientConnectionChanged() {
 
 void ConversationHandler::OnConversationTitleChanged(std::string title) {
```

Comment:
could this take a `const std::string&` or a `string_view`? It's not doing anything async